tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, test_context):
     def fail_broker_type(self, failure_mode, broker_type):
         # Pick a random topic and bounce it's leader
         topic_index = randint(0, len(self.topics.keys()) - 1)
-        topic = self.topics.keys()[topic_index]
+        topic = list(self.topics.keys())[topic_index]
```

Comment:
alternatively:
`topic = random.choice(list(self.topics.keys()))`

Reply:
I tried to do minimal changes and just fix the syntax difference between python2 and python3.
Let's keep it to simplify ongoing reviews?